Description: Update cmake rules for 'common' and 'thelib'. 
 Adds installation of the header files.

--- a/common/CMakeLists.txt
+++ b/common/CMakeLists.txt
@@ -27,3 +27,7 @@ IF(${LIB_TYPE} MATCHES "SHARED")
 	INSTALL(TARGETS ${PROJECT_NAME} LIBRARY DESTINATION ${CRTMPSERVER_ROOT})
 ENDIF(${LIB_TYPE} MATCHES "SHARED")
 
+INSTALL(DIRECTORY ${CRTMPSERVER_SOURCES_ROOT}/${PROJECT_NAME}/include/
+	DESTINATION ${CMAKE_INSTALL_PREFIX}/include/crtmpserver/${PROJECT_NAME}
+	FILES_MATCHING PATTERN "*.h")
+
--- a/thelib/CMakeLists.txt
+++ b/thelib/CMakeLists.txt
@@ -31,3 +31,6 @@ IF(${LIB_TYPE} MATCHES "SHARED")
 	INSTALL(TARGETS ${PROJECT_NAME} LIBRARY DESTINATION ${CRTMPSERVER_ROOT})
 ENDIF(${LIB_TYPE} MATCHES "SHARED")
 
+INSTALL(DIRECTORY ${CRTMPSERVER_SOURCES_ROOT}/${PROJECT_NAME}/include/
+	DESTINATION ${CMAKE_INSTALL_PREFIX}/include/crtmpserver/${PROJECT_NAME}
+	FILES_MATCHING PATTERN "*.h")
